compiler/il/OMRDataTypes.hpp

```diff
@@ -388,10 +414,13 @@ class OMR_EXTENSIBLE DataType
    int32_t getMaxPrecisionFromType();
 
    TR::DataType getVectorIntegralType();
-   TR::DataType getVectorElementType();
+   inline TR::DataType getVectorElementType();
+   inline TR::VectorLength getVectorLength();
+   inline static TR::DataTypes createVectorType(TR::DataTypes elementType, TR::VectorLength length);
+   inline static TR::VectorLength bitsToVectorLength(int32_t bits);
```

Comment:
Please provide Doxygen comments for these APIs.

Reply:
I thought names are pretty descriptive and the doxygen comments will make the code even a bit less readable. When all these methods are closely grouped together I think one can get a good idea on how to create and manipulate vector types. But, again, I can easily add those of course.

Reply:
The established precedent for new or changed APIs in the OMR compiler has been to create Doxygen comments.